libsql-sqlite3/src/wal.c

```diff
@@ -644,7 +659,7 @@ static int walIndexPage(
 /*
 ** Return a pointer to the WalCkptInfo structure in the wal-index.
 */
-static volatile WalCkptInfo *walCkptInfo(Wal *pWal){
+static volatile WalCkptInfo *walCkptInfo(sqlite3_wal *pWal){
```

Comment:
same for all s/Wal/sqlite3_wal below. We can change it later, but let's make it a separate patch to minimize the pains of rebasing and reverting if we ever need to. Keeping it as `Wal` also makes it easier to compare to upstream SQLite code